agent/engine/dockerclient/versionsupport_windows.go

```diff
@@ -11,15 +12,21 @@
 // express or implied. See the License for the specific language governing
 // permissions and limitations under the License.
 
-package functional_tests
+package dockerclient
 
-import (
-	"testing"
+func getAgentVersions() []DockerVersion {
+	return []DockerVersion{
+		Version_1_17,
+		Version_1_18,
+		Version_1_19,
+		Version_1_20,
+		Version_1_21,
+		Version_1_22,
+		Version_1_23,
+		Version_1_24,
+	}
+}
 
-	_ "github.com/aws/amazon-ecs-agent/agent/ecs_client/model/ecs"
-	_ "github.com/aws/aws-sdk-go/service/ec2"
-)
-
-// go test seems to require at least one non-tagged test to exist even when
-// running with a build tag
-func TestTest(t *testing.T) {}
+func getDefaultVersion() DockerVersion {
+	return Version_1_24
```

Comment:
You're right. I didn't do the other things mentioned on #687, I can address that separately